doc/nrf/releases/release-notes-2.4.0.rst

```diff
@@ -0,0 +1,895 @@
+    * The library now uses defines from the :file:`nrf_cloud_pgps.h` file for omitting parameters from a P-GPS request.
+      Removed the following values:
+      
+      * ``NRF_CLOUD_REST_PGPS_REQ_NO_COUNT``
+      * ``NRF_CLOUD_REST_PGPS_REQ_NO_INTERVAL``
+      * ``NRF_CLOUD_REST_PGPS_REQ_NO_GPS_DAY``
+      * ``NRF_CLOUD_REST_PGPS_REQ_NO_GPS_TOD``.
+
+    * A-GPS request encoding now uses the common codec function and new nRF Cloud API format.
+
+* :ref:`lib_lwm2m_client_utils` library:
+
+  * Added support for the connection pre-evaluation feature using the Kconfig option :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_LTE_CONNEVAL`.
+  * Updated the file :file:`lwm2m_client_utils.h`, which includes new API for FOTA to register application callback to receive state changes and requests for the update process.
+  * Removed the old API ``lwm2m_firmware_get_update_state_cb()``.
+
+* :ref:`lib_download_client` library:
+
+  * Updated:
+  
+    * The :c:func:`download_client_connect` function has been refactored to :c:func:`download_client_set_host` and made it non-blocking.
+    * The configuration from one security tag to a list of security tags.
+    * The library reports error ``ERANGE`` when HTTP range is requested but not supported by server.
+  
+  * Added:
+  
+    * The :c:func:`download_client_get` function that combines the functionality of functions :c:func:`download_client_set_host`, :c:func:`download_client_start`, and :c:func:`download_client_disconnect`.
+  
+
+  * Removed functions :c:func:`download_client_pause` and :c:func:`download_client_resume`.
+
+* :ref:`lib_lwm2m_location_assistance` library:
+
+  * Updated:
+
+    * :file:`lwm2m_client_utils_location.h` includes new API for location assistance to register application callback to receive result codes from location assistance.
+    * :file:`lwm2m_client_utils_location.h` by removing deprecated confirmable parameters from location assistance APIs.
+
+* :ref:`pdn_readme` library:
+
+  * Added the event ``PDN_EVENT_NETWORK_DETACH`` to indicate a full network detach.
+
+Libraries for NFC
+-----------------
+
+|no_changes_yet_note|
+
+Other libraries
+---------------
+
+* Added the :ref:`fem_al_lib` library for use in radio samples.
+
+* :ref:`dk_buttons_and_leds_readme` library:
+
+  * Updated the library to supports using the GPIO expander for the buttons, switches, and LEDs on the nRF9160 DK.
+
+* :ref:`app_event_manager` library:
+
+  * Added :c:macro:`APP_EVENT_ID` macro.
+
+* :ref:`event_manager_proxy` library:
+
+  * Removed the ``remote_event_name`` argument from the :c:func:`event_manager_proxy_subscribe` function.
+
+* :ref:`mod_memfault`:
+
+  * Added support for the ETB trace to be included in coredump.
+
+Common Application Framework (CAF)
+----------------------------------
+
+|no_changes_yet_note|
+
+Shell libraries
+---------------
+
+|no_changes_yet_note|
+
+Libraries for Zigbee
+--------------------
+
+|no_changes_yet_note|
+
+sdk-nrfxlib
```

Comment:
PR https://github.com/nrfconnect/sdk-nrf/pull/11293 will affect this section

Reply:
Added